region/brinstar/red/Beta Power Bomb Room.json

```diff
@@ -522,7 +522,11 @@
       "name": "Damage Boost Escape",
       "requires": [
         {"obstaclesCleared": ["B"]},
-        "canHorizontalDamageBoost",
+        "canNeutralDamageBoost",
+        {"or": [
+          "canMidAirMorph",
+          "canTrickyJump"
```

Comment:
With trickyjump, you would land on the safe spot using #21

Reply:
Right, with this PR the morphless damage boost option doesn't add anything logically since it is technically superseded by the safe spot strat; having it as an option just helps document that it's a possible thing to do. This is assuming VH is the right place for both the morphless damage boost and safe spot strats?

Reply:
We typically havent been including strats that are strictly harder or have more requirements than another strat (This seems to be a similar difficulty but has more requirements). Generally, when we add strats like that, we will try to combine them together, so that we can mention them both in the same note - which could help provide an alternative if someone is struggling with a strat. Is there a good way to combine the morphless tricky jump variant with the safe spot jump?